{"name": "测试数据：【 1532341，111111a，{except_result}，10000，token】", "status": "broken", "statusDetails": {"message": "AttributeError: 'Response' object has no attribute 'jsonpath'", "trace": "self = <testcases.api_test.test_03_login.TestUserLogin object at 0x000001B5F8D7E358>\nusername = '1532341', password = '111111a', login_type = 1, except_code = 10000\nexcept_msg = 'token'\n\n    @allure.story(\"用例--登录用户\")\n    @allure.description(\"该用例是针对获取用户登录接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{password}，{except_result}，{except_code}，{except_msg}】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, password, login_type, except_code, except_msg\",\n                             api_data[\"test_login_user\"])\n    def test_login_user(self, username, password, login_type, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n>       result = login_user(username, password, login_type)\n\ntest_03_login.py:28: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nusername = '1532341', password = '111111a', login_type = 1\n\n    def login_user(username, password, login_type):\n    \n        \"\"\"\n        登录用户\n        :param username: 用户名\n        :param password: 密码\n        :return: 自定义的关键字返回结果 result\n        \"\"\"\n        result = ResultBase()\n        payload = {\n            \"username\": username,\n            \"password\": password,\n            \"login_type\": login_type\n        }\n        header = {\n            \"Content-Type\": \"application/json\"\n        }\n        res = user.login(json=payload, headers=header)\n        logger.info(\"登录用户 ==>> 返回结果 ==>> {}\".format(res))\n        if \"token\" in res.text:\n>           token = res.jsonpath(res.json(),'$..token')\nE           AttributeError: 'Response' object has no attribute 'jsonpath'\n\n..\\..\\operation\\user.py:101: AttributeError"}, "description": "该用例是针对获取用户登录接口的测试", "attachments": [{"name": "log", "source": "5c69f739-870f-4955-ad20-194f99b39895-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'1532341'"}, {"name": "password", "value": "'111111a'"}, {"name": "login_type", "value": "1"}, {"name": "except_code", "value": "10000"}, {"name": "except_msg", "value": "'token'"}], "start": 1669190383168, "stop": 1669190384737, "uuid": "9abd9c34-53ea-4a76-b840-7905bd6d711f", "historyId": "fc744133cd1f19c331d93737caf7e646", "testCaseId": "7272db244dabef390096479ae3196642", "fullName": "testcases.api_test.test_03_login.TestUserLogin#test_login_user", "labels": [{"name": "story", "value": "用例--登录用户"}, {"name": "feature", "value": "用户登录模块"}, {"name": "severity", "value": "normal"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_03_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "lijuan"}, {"name": "thread", "value": "21996-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_03_login"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}